include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -49,6 +49,26 @@ class OpKernel {
     ORT_NOT_IMPLEMENTED(__FUNCTION__, " is not implemented");
   }
 
+  // Override this function to PrePack the constant Tensor to the format as needed.
+  // For example, MatMul kernel can pack the input B if it is constant like code below.
+  //   Status PrePack(const Tensor& tensor, int input_idx, bool& is_packed) override {
+  //     is_packed = false;
```

Comment:
Slightly confusing to say 'if it is constant' given 'tensor' is always for a constant initializer.